back/src/main/resources/properties/jwt.yml

```
@@ -0,0 +1,14 @@
+  # local 환경
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)